scripts/unreleasedChanges.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
```

Comment:
I was getting a lot of grief trying to run this in a worker using `ts-node` so I reverted back to javascript and made it an executable script